pyproject.toml

```
@@ -45,6 +45,7 @@ copier = "copier.__main__:CopierApp.run"
 
 [dependency-groups]
 dev = [
+  "codespell==2.4.1",
```

We need to install the [`toml` extra](https://github.com/codespell-project/codespell/blob/ed6185a67247989111d989498d44c81390dab043/pyproject.toml#L50-L52), as codespell relies on `tomli` for Python <3.11 when sourcing the config from `pyproject.toml` (which I prefer, so all good).

```suggestion
  "codespell[toml]==2.4.1",
```

Don't forget to update the the lock file.